.circleci/config.yml

```diff
@@ -1,16 +1,16 @@
 version: 2
 jobs:
-  test_jdk8_with_coverage:
+  test_jdk11_with_coverage:
     working_directory: ~/styx
     docker:
-      - image: maven:3.5.4-jdk-8
+      - image: maven:3.6.0-jdk-11
     steps:
       - checkout
       - restore_cache:
-          key: v2-deps-{{ checksum "pom.xml" }}
+          key: v3-deps-{{ checksum "pom.xml" }}
       - run: mvn -B dependency:go-offline compile test-compile -P disable-static-analysis
```

Comment:
Don't remember why we do this. Is it because errorprone doesn't work well with java11?